resolwe/flow/executors/__init__.py

```diff
@@ -122,6 +123,14 @@ def run(self, data_id, script, verbosity=1):
 
         self.data_id = data_id
 
+        # Fetch data instance to get any executor requirements.
+        if data_id != 'no_data_id':
```

Comment:
I am not sure why this test is done like it is. I think it would be better to just use the `ProcessTestCase` which uses actual data objects (something like I do in the new `test_executor_requirements`). @dblenkus?